### DISCUSSION TITLE
Permission Tracking

### DISCUSSION THREAD

**Discussion by jrhaberland** [simple-icons/simple-icons#11269] [Ideas]
I'm pretty sure this doesn't exist in the repository as I've looked pretty thoroughly for it. I feel like there should be some place that keeps a log of the companies that have given simple-icons permission to use their icons.

If it were to be made it should store the following:
- Name of company
- Date permission was granted
- A copy of the email that gave said permission
- Link to related issues and or PRs
- Maybe a list of all the icons related with the company

**Comment by uncenter** [comment]
All permission requests and approvals can be found in the pull request where it was first necessary to do so, I think this works fine for our needs at the moment as we have no use for the data in our automated workflows nor is it hard to find as I just mentioned. For *forbidden* brands though I think we are planning to store data around those: https://github.com/simple-icons/simple-icons/issues/9999.

**Reply by jrhaberland** [reply]
That is true however when one brand owns quite a few sub-brands would be nice to have one spot to go and see if that brand has given permission for their other logos. I started thinking about this because of #11265. That way it would be easy to determine that if GM approves an icon, they'd probably be okay with their other icons being added as well. Instead of having to search through each of their brands issues/PRs to find the one where they gave permission.

**Reply by uncenter** [reply]
Would every single instance of permission being granted be tracked? How will we know if a brand will/has more products to be added at some point?

**Reply by jrhaberland** [reply]
I'd imagine that every instance where permission is given would be logged, at least in a .md file or something. Just to have something to easily look back on. It's mostly helpful for bigger brands like Google, Microsoft, Cloudflare, GM, Apple who each have quite a few logos.

**Reply by uncenter** [reply]
Probably a .json -> autogenerated markdown I'd imagine. This sounds good though!

**Reply by jrhaberland** [reply]
Yeah that would work

**Reply by uncenter** [reply]
In case you didn't notice I've created an issue out of this discussion at https://github.com/simple-icons/simple-icons/issues/11270.
